src/powerflow_method.jl

```diff
@@ -364,6 +364,9 @@ function _newton_powerflow(
         if data.calculate_loss_factors
             calculate_loss_factors(data, J.Jv, time_step)
         end
+        if data.calculate_voltage_stability_factors
+            calculate_voltage_stability_factors(data, J, time_step)
```

Comment:
Best to not have two different things both called `calculate_voltage_stability_factors`